.github/workflows/lint.yml

```diff
@@ -62,18 +62,12 @@ jobs:
 
       - name: Lint Changed Files
         run: |
-          jq -r '.[]' ${HOME}/files_modified.json ${HOME}/files_added.json | sort | uniq > /tmp/changed_files.txt
-          CHANGED_FILES=$(cat /tmp/changed_files.txt)
-          echo "These are the changed files: $CHANGED_FILES"
+          CHANGED_FILES=($(jq -r '.[]' ${HOME}/files_modified.json ${HOME}/files_added.json | grep '.\+.\(js\|ts\|tsx\)$' | sort -u))
```

Comment:
I remember one of the issues we were trying to tackle was when a file is renamed, both 'modified' and 'added' shas will show the same renamed file which lead to later lint that file more than once, is it? @RyanL1997 , is this considered in the new change as I saw uniq was removed?

Reply:
prefer `sort -u` over `sort | uniq`. they have the same output

Reply:
@mengweieric Yes, that shouldn't be a problem. Since the above step `Check Changes of Files` has filtered the output